data/isdv4-51b0.tablet

```diff
@@ -0,0 +1,21 @@
+# Lenovo ThinkPad X13 Yoga
+# Sensor Type: AES
+# Features: Touch (Integrated), Tilt
+# HW Resolution: 29376 x 16524 (2540 x 2540 lpi)
+#
+# Autogenerated from sysinfo.wym3nbQqZN
```

Comment:
I think we should reference where we can find this, otherwise it just looks like a leftover.

Reply:
Would a link to https://github.com/linuxwacom/wacom-hid-descriptors/ suffice? I really don't want to have to dynamically generate URLs.

Reply:
Added a link; re-open this thread if that's not enough

Reply:
yes, sorry, email got drowned out. That link is sufficient, thanks.